# Kunskapskontroll 1: Statistik och dataanalys

## Del 1 - Teoretiska frågor

### Fråga 1:
Många barn tror att de blev till genom att en stork kom och lämnade dem till deras föräldrar.  
 
För att undersöka det kan vi kolla på data och det finns en graf enligt figuren nedan.  
 
Det verkar alltså finnas ett samband mellan antalet storkar och barnafödsel!? Stämmer detta påståendet? Använd begreppen ”korrelation” och ”kausalitet” i ditt svar.  

Nej, påståendet att antalet storkar orsakar fler barnafödselar stämmer inte, även om det kan verka så vid en första anblick. 

Korrelation och kausalitet:

Korrelation innebär att två variabler verkar följa varandra – till exempel att i områden där det finns fler storkar, föds också fler barn.

Kausalitet innebär att den ena variabeln direkt påverkar den andra - att storkarna orsakar att fler barn föds.

I det här fallet finns det korrelation, men ingen kausalitet. Bara för att två saker sker samtidigt betyder det inte att den ena orsakar den andra. Att det finns en korrelation mellan antalet storkar och barnafödsel betyder inte att det finns en kausalitet.


### Fråga 2: Vad är korrelation

Korrelation mäter hur starkt två variabler hänger ihop - att om det finns ett mönster i hur de förändras tillsammans.

I bilden ses en rad diagram med olika siffror – till exempel 1, 0.8, 0, -1. Dessa siffror kallas korrelationskoefficienter (eller "r") och de går från -1 till +1:

Positiv korrelation (r > 0)
r = 1: Perfekt positiv korrelation – när X ökar, ökar Y exakt lika mycket. Punkterna ligger på en rak uppåtgående linje.

r = 0.8: Stark positiv korrelation – punkterna ligger nära en stigande linje.

r = 0.4: Svagare positiv korrelation – viss spridning, men tydlig trend uppåt.

Ingen korrelation (r = 0)
r = 0: Ingen linjär koppling mellan X och Y. Punkterna är spridda utan mönster.

Negativ korrelation (r < 0)
r = -1: Perfekt negativ korrelation – när X ökar, minskar Y exakt lika mycket. Rak nedåtgående linje.

r = -0.8: Stark negativ korrelation – tydlig trend nedåt, men med lite spridning.

r = -0.4: Svag negativ korrelation – lite mindre tydligt men ändå en nedåtgående trend.


### Fråga 3:

Kim påstår att ”medelvärdet” är ett bättre mått än ”medianvärdet”.

Jag håller inte med Kim. Medelvärdetn är inte ett bättre mått än medianvärdet eftersom det beror på situation när vi behöver dessa mått.

Medelvärde är summan av alla värden ÷ antal värden
Exempel: 2, 4, 6 → medelvärde = (2 + 4 + 6)/3 = 4

Medianvärde är det mittersta värdet när alla siffror är i ordning
Exempel: 2, 4, 6 → median = 4
Om det är ett jämnt antal värden, tar man medel av de två mittersta.

I situation där vi kollar på lönerna i en avdelning som är jämnt fördelad mellan 25 000 till 30 000 då är det bättre att använda medelvärdet medans om det finns en person (manager) som tjänar 200 000 och andra tjänar bara 30 000, blir medianvärdet bättre att använda. 



### Fråga 4: 

Vad används cirkeldiagram för? Ge ett exempel på vad ett företag som Spotify (eller
något annat företag som du själv väljer) hade kunnat använda cirkeldiagram för. 

Cirkeldiagram används för att visa hur en helhet är uppdelad i delar –  hur olika kategorier bidrar till en totalsumma. Det är bra när visualiseringar vill visa andelar eller procent på ett tydligt och visuellt sätt.

Spotify kan använda ett cirkeldiagram för att visa:

Fördelning av musiklyssning per genre

T.ex. hur stor andel av användarnas lyssningar

Pop: 40%

Hip-hop: 25%

Rock: 15%

Elektronisk musik: 15%

Jazz: 3%

Övrigt: 2%

Totalt ska det vara 100%


### Fråga 5:


Linjediagram används för att visa förändringar över tid. Det funkar bäst när man vill följa en utveckling – t.ex. dag för dag, vecka för vecka eller år för år.

Spotify kan använda ett linjediagram för att visa:

Antal nya användare per månad under ett år

Januari: 400 000 accounts

Februari: 405 000 accounts

Mars: 415 000 accounts

...

December: 450 000 accounts

Diagrammet visar då trenden i hur användarantalet förändras över tid – om det ökar, minskar eller står still.



### Fråga 6:


Lådagram, eller boxplot, används för att visa hur data är fördelad –  spridningen i en datamängd. Det hjälper att snabbt se:

Lägsta värde (minimum)

Första kvartilen (Q1)

Medianen (Q2)

Tredje kvartilen (Q3)

Högsta värde (maximum)

Eventuella outliers (extremvärden)

Lådagram ska fungera bäst när man vill 

- Jämföra fördelningen av olika grupper.

- Se hur symmetrisk eller skev en datamängd är.

- Identifiera avvikande värden.

- Få en tydlig överblick över hur spridda värdena är.



### Fråga 7:


Medelvikten = 3.5 kg

Detta betyder att de flesta nyfödda väger underfär 3.5 kg

Standardavvikelsen = 0.5 kg

Det innebär att 

- Cirka 68 % väger mellan 3.0 kg och 4.0 kg (±1 standardavvikelse)

- Cirka 95 % väger mellan 2.5 kg och 4.5 kg (±2 standardavvikelser)

- Cirka 99.7 % väger mellan 2.0 kg och 5.0 kg (±3 standardavvikelser)





In [ ]:
from scipy.stats import norm

medelvarde = 3.5
standardavvikelse = 0.5

# Sannolikheten att ett barn väger över 4.5 kg
sannolikhet_over_4_5 = 1 - norm.cdf(4.5, medelvarde, standardavvikelse)
print(f"Sannolikheten att ett barn väger över 4.5 kg är: {sannolikhet_over_4_5:.4f}")

# Sannolikheten att ett barn väger mindre än 3 kg
sannolikhet_under_3 = norm.cdf(3, medelvarde, standardavvikelse)
print(f"Sannolikheten att ett barn väger mindre än 3 kg är: {sannolikhet_under_3:.4f}")

# Förväntat antal barn över 4.5 kg bland 1000 barn
antal_barn = 1000
forvantat_antal_over_4_5 = sannolikhet_over_4_5 * antal_barn
print(f"Förväntat antal barn som väger över 4.5 kg bland {antal_barn} barn är: {forvantat_antal_over_4_5:.2f}")

Sannolikheten att ett barn väger över 4.5 kg är: 0.0228
Sannolikheten att ett barn väger mindre än 3 kg är: 0.1587
Förväntat antal barn som väger över 4.5 kg bland 1000 barn är: 22.75


Sannolikheten att ett barn väger över 4.5 kg är: 0.0228

Sannolikheten att ett barn väger mindre än 3 kg är: 0.1587

Förväntat antal barn som väger över 4.5 kg bland 1000 barn är: 22.75 (23 barn)

